dvc/repo/experiments/queue/base.py

```diff
@@ -131,15 +131,27 @@ def put(self, *args, **kwargs) -> QueueEntry:
     def get(self) -> QueueGetResult:
         """Pop and return the first item in the queue."""
 
-    def remove(self, revs: Collection[str]) -> List[str]:
+    def remove(
+        self,
+        revs: Collection[str],
+        _all: bool = False,
```

Comment:
```suggestion
        all_: bool = False,
```

looks like this one was missed